pvlib/shading.py

```diff
@@ -232,3 +232,64 @@ def sky_diffuse_passias(masking_angle):
        Available at https://www.nrel.gov/docs/fy18osti/67399.pdf
     """
     return 1 - cosd(masking_angle/2)**2
+
+
+def projected_solar_zenith_angle(surface_tilt, surface_azimuth,
+                                 solar_apparent_elevation, solar_azimuth):
+    r"""
+    Calculate projected solar zenith angle in degrees.
+
+    This is common in track and shadow computation [1]_ [2]_ [3]_.
+
+    Parameters
+    ----------
+    surface_tilt : numeric
+        Axis tilt angle in degrees. From horizontal plane to array plane.
+    surface_azimuth : numeric
```

Comment:
All of the `surface_*` in this function should be `axis_*`; `surface` refers to orientation of a PV module while `axis` refers to the rotation axis around which a module rotates.